scratch3/blocks.js

```diff
@@ -91,6 +91,7 @@ IconView.prototype.draw = function() {
 
 IconView.icons = {
   greenFlag: { width: 20, height: 21, dy: -2 },
+  stopSign: { width: 20, height: 20, dx: 1, dy: -5 },
```

Comment:
To me it looks like this is too high:

<img src="https://user-images.githubusercontent.com/639289/132942254-c9d2b7e5-1a46-4197-89c6-5d990d660027.png" width=200>

I think we want
```suggestion
  stopSign: { width: 20, height: 20 },
```

Reply:
I've removed the offsets, but now the stop sign is too low on stack blocks:
![this](https://user-images.githubusercontent.com/68464103/132944869-875ae917-6a89-495e-9782-422c602fce67.png)

Reply:
Interesting! That appears to be a bug which is only triggered when there is no text in the block? It's hard to show in a screenshot, but when I add label text the y-height of the icon seems to change.

We might want to fix that, but it can be a separate PR!